src/main/java/page_objects/MyTicketPage.java

```diff
@@ -1,4 +1,33 @@
 package page_objects;
 
+import helpers.DriverHelper;
+import helpers.ElementHelper;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
 public class MyTicketPage extends BasePage {
+
+    //Locators
+    private By btnCancel = By.cssSelector("input[value='Cancel']");
+    private By rowTicket = By.xpath("//table[@class='MyTable']/tbody//following-sibling::tr");
+
+    //Elements
+    private WebElement getBtnCancel() {
+        return DriverHelper.getDriver().findElement(btnCancel);
+    }
+
+    //Methods
+    public void clickCancelButon() {
+        getBtnCancel().click();
+    }
+
+    public void cancelTicket() {
+        ElementHelper.scrollToView(getBtnCancel());
+        clickCancelButon();
+        DriverHelper.getDriver().switchTo().alert().accept();
```

Comment:
wrap this to DriverHelper

Reply:
I created new acceptAlert() method in DriverHelper class